modules/rostests/winetests/user32/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 
+remove_definitions(-DWINVER=0x502 -D_WIN32_WINNT=0x502)
+add_definitions(-DWINVER=0x600 -D_WIN32_WINNT=0x600)
+
+#if (_WIN32_WINNT >= 0x0600)
```

Comment:
?

Reply:
Hmm, I'm not sure how I ended up with this line, probably accidental paste :)